pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[tool.poetry]
+name = "udata-front"
+version = "2.0.9-alpha.0"
+description = "udata customizations for data.gouv.fr"
+readme="README.md"
+authors = ["Etalab <pypi@data.gouv.fr>"]
+repository="https://github.com/etalab/udata-front"
+license = "LGPL"
+keywords = ["udata", "opendata", "portal", "etalab"]
+classifiers=[
+    "Development Status :: 3 - Alpha",
+    "Programming Language :: Python",
+    "Environment :: Web Environment",
+    "Operating System :: OS Independent",
+    "Intended Audience :: Developers",
+    "Topic :: System :: Software Distribution",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "License :: OSI Approved :: GNU Library or Lesser General Public License (LGPL)"
+]
+
+[tool.poetry.dependencies]
+python = ">=3.7,<4"
+feedparser = "6.0.8"
+python-frontmatter = "1.0.0"
+sentry-sdk = {version = "1.1.0", extras = ["flask"]}
+udata = { git = "https://github.com/opendatateam/udata.git", rev = "poetrySwitch" }
```

Comment:
Don't forget to remove the branch when merging https://github.com/opendatateam/udata/pull/2757